### PR TITLE
Feature/polishing build work

### PR DESCRIPTION
**Pull request by jisikoff**
Build works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,100 @@
...
+    end
+    ```
+
+## Configuring the Adapter
+
+```elixir
+# config/config.exs
+
+config :my_app, MyApp.Broker,
+  adapter: ConduitMQTT, connection_opts: [server: {Tortoise.Transport.Tcp, host: 'localhost', port: 1883}]  #TODO IS THIS CORRECT WAY TO GET INTO ADAPTER OPTS?
```
Comment (blatyo) [inline README.md]: Yes. Add some newlines.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,100 @@
...
+
+# Stop lager redirecting :error_logger messages
+config :lager, :error_logger_redirect, false
+
+# Stop lager removing Logger's :error_logger handler
+config :lager, :error_logger_whitelist, [Logger.ErrorHandler]
```
Comment (blatyo) [inline README.md]: These two lager things should be unnecessary unless tortoise is bringing it in. The rabbit stuff does, which is why it was there.

File: README.md [hunk 3]
```diff
@@ -1,20 +1,100 @@
...
 - Keyword lists in the GenServer's should probably be maps or normal lists
-- building on circleci
-- add the badges to the readme
-- add a changelog
-- add credo
-- add dialyxir
-- make sure it's formatted
-- add this alias and the corresponding function?https://github.com/conduitframework/conduit/blob/master/mix.exs#L34
\ No newline at end of file
+- add the badges to the readme
```
Comment (blatyo) [inline README.md:126]: We should document somewhere visible that MQTT doesn't support headers natively and that wrapping/unwrapping is necessary.

File: lib/conduit_mqtt/event_logger.ex [hunk 4]
```diff
@@ -1,6 +1,6 @@
 defmodule ConduitMQTT.EventLogger do
   @moduledoc """
-  Manages an Tortoise connection
+  Can be used to log Tortoise Events (NOT USED RIGHT NOW)
```
Comment (blatyo) [inline lib/conduit_mqtt/event_logger.ex]: This should probably go away

File: lib/conduit_mqtt/plug/unwrap.ex [hunk 5]
```diff
@@ -1,4 +1,7 @@
 defmodule ConduitMQTT.Plug.Unwrap do
+  @moduledoc """
+  Plug to help unwrap headers and attributes from the MQTT body payload
```
Comment (blatyo) [inline lib/conduit_mqtt/plug/unwrap.ex:3]: This should have more docs and probably be linked to from the README

File: lib/conduit_mqtt/plug/wrap.ex [hunk 6]
```diff
@@ -1,4 +1,7 @@
 defmodule ConduitMQTT.Plug.Wrap do
+  @moduledoc """
+  Plug to help wrap headers and attributes into the MQTT body payload
```
Comment (blatyo) [inline lib/conduit_mqtt/plug/wrap.ex:3]: Same

File: README.md (hunk 1 above)
Comment (jisikoff) [reply README.md]: where are newlines valid?  I need the connection_opts to get passed in the opts of the ConduitMQTT adapter not anything higher up that list there

**Pull request merged**